common/src/test/java/com/revenuecat/purchases/common/verification/SigningManagerTest.kt

```diff
@@ -203,13 +206,17 @@ class SigningManagerTest {
     private fun callVerifyResponse(
         signingManager: SigningManager,
         requestPath: String = "test-url-path",
-        responseCode: Int = RCHTTPStatusCodes.SUCCESS,
-        signature: String? = "2bm3QppRywK5ULyCRLS5JJy9sq+84IkMk0Ue4LsywEp87t0tDObpzPlu30l4Desq9X65UFuosqwCLMizruDHbKvPqQLce0hrIuZpgic+cQ8=",
+        // Generated signature using test keys:
+        // Test root private key: YMHMQMpepBKamtSzO8KCN2M8Z3AUW5R1JXIFtxUWFUI
+        // Test root public key: yg2wZGAr8Af+Unt9RImQDbL7qA81txk+ga0I+ylmcyo=
+        // Test intermediate private key: fPBoIjQ7DecE89ATW6PZsqLVQNyEs5fiX3sUyS3U4YI
+        // Test intermediate public key: xoDYyUeHnIlSIAeOOzmvdNPOlbNSKK+xE0fE/ufS1fs=
```

Comment:
I wasn't sure whether to add these, but I thought it might be useful if we need to debug things. Also, they are test keys, not real keys so I think it should be fine.

Reply:
Yeah.